tests/unit/test_ffmpeg.py

```diff
@@ -103,8 +103,8 @@ def test_frames(video_with_small, start, end, step, error):
         ff_frames(**arguments)
 
         # Check that progress updates are complete
-        expected_file_no = ((end - start) / step) + 1
-        assert frame_indices == range(1, expected_file_no + 1)
+        expected_file_no = int(((end - start) / step) + 1)
```

Comment:
`type(expected_file_no) == float`, which will raise a `TypeError` when passed as an argument to `range`